public/index.html

```diff
@@ -0,0 +1,171 @@
+        </div>
+        <div class="feature-list-card-container container">
+          <div class="feature-list-img-card">
+            <img class="card-img" src="/public/assets/1.jpg" alt="house 1" />
+            <h3 class="card-title">東京都XX様宅</h3>
+            <p class="card-description">ここに説明を入れます。サンプルテキスト。</p>
+            <button class="card-btn btn-1">詳しくみる</button>
+          </div>
+          <div class="feature-list-img-card">
+            <img class="card-img" src="/public/assets/2.jpg" alt="house 1" />
+            <h3 class="card-title">東京都XX様宅</h3>
+            <p class="card-description">ここに説明を入れます。サンプルテキスト。</p>
+            <button class="card-btn btn-2">詳しくみる</button>
+          </div>
+          <div class="feature-list-img-card">
+            <img class="card-img" src="/public/assets/3.jpg" alt="house 1" />
+            <h3 class="card-title">東京都XX様宅</h3>
+            <p class="card-description">
+              ここに説明を入れます。サンプルテキスト。ここに説明を入れます。サンプルテキスト。ここに説明を入れます。サンプルテキスト。
+            </p>
+            <button class="card-btn">詳しくみる</button>
+          </div>
+        </div>
+      </section>
+      <section class="commerce-info container-commerce container section">
+        <div class="whats-new">
+          <h1>新着情報</h1>
+          <hr />
+          <div class="commerce-info-row">
+            <time class="commerce-dates" datetime="2023-11-17">2023/11/17</time>
+            <div class="commerce-tags tag1">お知らせ</div>
+            <p class="commerce-description">
+              ハウスメーカー・工務店向け無料ホームページテンプレート tp_housebuilder1公開。
+            </p>
+          </div>
+          <div class="commerce-info-row">
+            <time class="commerce-dates" datetime="2023-11-17">20XX/00/00</time>
+            <div class="commerce-tags tag2">施工事例</div>
+            <p class="commerce-description">サンプルテキスト。サンプルテキスト。サンプルテキスト。</p>
+          </div>
+          <div class="commerce-info-row">
+            <time class="commerce-dates" datetime="2023-11-17">20XX/00/00</time>
+            <div class="commerce-tags tag3">キャンペーン</div>
+            <p class="commerce-description">サンプルテキスト。サンプルテキスト。サンプルテキスト。</p>
+          </div>
+          <div class="commerce-info-row">
+            <time class="commerce-dates" datetime="2023-11-17">20XX/00/00</time>
+            <div class="commerce-tags tag4">お知らせ</div>
+            <p class="commerce-description">サンプルテキスト。サンプルテキスト。サンプルテキスト。</p>
+          </div>
+          <div class="commerce-info-row">
+            <time class="commerce-dates" datetime="2023-11-17">20XX/00/00</time>
+            <div class="commerce-tags tag5">施工事例</div>
+            <p class="commerce-description">
+              サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。
+            </p>
+          </div>
+        </div>
+        <div class="template-info">
+          <h1>テンプレートのご利用前に必ずお読み下さい</h1>
+          <hr />
+          <h3>利用規約のご案内</h3>
```

Comment:
[Must]
The tag must be h2. Because the heading tags are supposed to be consecutively.
You can see [the examples](https://www.w3.org/WAI/tutorials/page-structure/headings/) on the W3C article page.
And the heading tags have a parent tag which is  `section` tag. So the area that is named `template-info` will be like this.

```.html
<section class="template-info">
    <h1 class="template-info-heading"></h1>
    <p class="paragraph"></p>
    ...
</section>
```

The heading tag doesn't have to be coded right under the section tag like the example above. We often need to wrap the tag with `div` for styling.

Reply:
In case what I said is not correctly conveyed. 

<img width="599" alt="Screenshot 2024-03-13 at 18 59 35" src="https://github.com/minierparedes/housebuilder/assets/156676782/ec3c1d52-25a0-43f1-b65c-6388d57feeb1">

Hint: HTML is a language that tells what does mean of the page. Tags are the meaning of the page.
To know about it more, coding a book in HTML would help your understanding.


---
The most painful and difficult part of HTML is how you are aware of the meaning of the sentence the website conveys.

Another example from books:

<img width="1082" alt="Screenshot 2024-03-13 at 19 11 57" src="https://github.com/minierparedes/housebuilder/assets/156676782/ee320b36-07e5-43bd-bfc6-343ed1a23ad3">

<img width="538" alt="Screenshot 2024-03-13 at 19 16 59" src="https://github.com/minierparedes/housebuilder/assets/156676782/f5d3d60f-6aaa-4821-bbd2-875603143776">

Am I correct? Who knows.

Reply:
@sharray2918 
Thank you for providing me with this detail explanation.
I have made sure to further read into the topic and changed
the code base to reflect the issue raised.